build-docker.sh

```diff
@@ -1,27 +1,104 @@
 #!/usr/bin/env bash
-set -e
+set -e -o pipefail
 
-CONTAINER_NAME=trezor-firmware-env.nix
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+CONTAINER_NAME=${CONTAINER_NAME:-trezor-firmware-env.nix}
+ALPINE_VERSION=${ALPINE_VERSION:-3.12.0}
+ALPINE_RELEASE=${ALPINE_RELEASE:-3.12}
+ALPINE_ARCH=${ALPINE_ARCH:-x86_64}
+ALPINE_CDN=${ALPINE_CDN:-http://dl-cdn.alpinelinux.org/alpine}
+CONTAINER_FS_URL=${CONTAINER_FS_URL:-"$ALPINE_CDN/v$ALPINE_RELEASE/releases/$ALPINE_ARCH/alpine-minirootfs-$ALPINE_VERSION-$ALPINE_ARCH.tar.gz"}
+CACHES_VOLUME_NAME=${CACHES_VOLUME_NAME:-trezor_firmware_caches}
```

Comment:
Do we need these variables configurable? Why not just setting them?

Reply:
Personally I'm for optional configurability but all env config vars configurable from outside should have `TREZOR_` prefix to reduce risk of picking something from env accidentally. IMO, this should be standard "hygiene" when writing bash scripts.